sources/refund.move

```diff
@@ -131,100 +208,171 @@ module refund::refund {
         } else {
             none()
         }
     }
-    public fun funding(pool: &RefundPool): u64 { balance::value(&pool.funds) }
-    public fun total_refunded(pool: &RefundPool): u64 { pool.accounting.total_refunded }
-    public fun total_boosted(pool: &RefundPool): u64 { pool.accounting.total_boosted }
-    public fun current_liability(pool: &RefundPool): u64 { pool.accounting.current_liability }
-    
-    /// Calculates the current liability of the RefundPool, considering the boosted refund scenario.
-    ///
-    /// This function computes the total current liability of the pool and then adds an additional 50% to model
-    /// the scenario where all outstanding refunds are claimed with a 150% boost. This is useful for assessing
-    /// the potential maximum liability under boosted refund conditions.
-    public fun current_liability_boosted(pool: &RefundPool): u64 { pool.accounting.current_liability + div(pool.accounting.current_liability, 2) }
-    
-    /// Calculates the unfunded liability of the RefundPool.
-    ///
-    /// This function determines the difference between the current liabilities (the total amount the pool
-    /// needs to refund) and the available funds in the pool. If the available funds are sufficient to cover
-    /// all liabilities, the unfunded liability is zero. Otherwise, it represents the shortfall that must be
-    /// addressed to fully fund all refund claims.
-    public fun unfunded_liability(pool: &RefundPool): u64 {
-        let available = balance::value(&pool.funds);
-        if (available >= pool.accounting.current_liability) {
-            0
-        } else {
-            pool.accounting.current_liability - available
-        }
+    
+    public fun get_total_to_refund(pool: &RefundPool): u64 { total_to_refund(&pool.accounting) }
+    public fun get_total_raised(pool: &RefundPool): u64 { total_raised(&pool.accounting) }
+    public fun get_total_refunded(pool: &RefundPool): u64 { total_refunded(&pool.accounting) }
+    public fun get_total_boosted(pool: &RefundPool): u64 { total_boosted(&pool.accounting) }
+    public fun base_funds(pool: &RefundPool): u64 { balance::value(funds(&pool.base_pool)) }
+    public fun booster_funds(pool: &RefundPool): u64 { balance::value(funds(&pool.booster_pool)) }
+    public fun current_liabilities(pool: &RefundPool): u64 { accounting::current_liabilities(&pool.accounting) }
+
+    // === Mutators (Friends) ===
+    
+    public(friend) fun unclaimed_mut(pool: &mut RefundPool): &mut Table<address, u64> { &mut pool.unclaimed }
+    public(friend) fun accounting_mut(pool: &mut RefundPool): &mut Accounting { &mut pool.accounting }
+    public(friend) fun booster_pool_mut(pool: &mut RefundPool): &mut Pool { &mut pool.base_pool }
+
+    // === Phase Transitions ===
+
+    public entry fun start_timeout_setup_phase(
+        pub: &Publisher,
+        pool: &mut RefundPool,
+    ) {
+        assert_address_addition_phase(pool);
+        assert_publisher(pub);
+
```

Comment:
Is there an target (or at least a lower bound) for `total_to_refund`? Would be good to assert that here if so.